web/ci.sh

```diff
@@ -24,15 +24,22 @@ cd "$THIS_SCRIPT_PATH"
 
 builder_describe "Defines and implements the CI build steps for Keyman Engine for Web (KMW)." \
   "build" \
-  "test             Runs all unit tests."  \
-  "post-test        Runs post-test cleanup.  Should be run even if a prior step fails." \
-  "validate-size    Runs the build-size comparison check" \
-  "--debug          Runs this script in local-development mode; reports and tests will be locally logged"
+  "test                 Runs all unit tests."  \
+  "post-test            Runs post-test cleanup.  Should be run even if a prior step fails." \
+  "validate-size        Runs the build-size comparison check" \
+  "publish-s.keyman     Prepares an s.keyman.com PR (intended for release builds)" \
+  "publish-downloads    Prepares the upload to downloads.keyman.com (intended for release builds)" \
+  "--debug              Runs this script in local-development mode; reports and tests will be locally logged" \
+  "--password=PASSWORD  Used to supply passwords needed by certain actions"
 
 builder_parse "$@"
 
 ####
 
+TIER=`cat ../TIER.md`
+BUILD_NUMBER=`cat ../VERSION.md`
```

Comment:
You already sourced build-utils.sh, so isn't this already available as $VERSION?

And is BUILD_NUMBER meant to be the full version, or a build counter? The latter is an environment variable `env.build_counter` on the CI

Reply:
Ah, I suppose I could simplify there re `$VERSION`.  The original PowerShell script did not have that access.  (Compare to Web's "Release - 14.0" config, steps 3 and 4.)